examples/gcf-pubsub-vm-delete-event-handler/main.py

```diff
@@ -0,0 +1,660 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+__author__ = ('Jeff McCune <jeff@openinfrastructure.co>, '
+              'Gary Larizza <gary@openinfrastructure.co>')
+
+import base64
+import json
+import logging
+import os
+import google.cloud.logging
+from enum import Enum
+from google.cloud.logging.resource import Resource
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+from typing import List
+
+
+class RuntimeState:
```

Comment:
Confused to the value of this empty class.

Reply:
It's intended to hold the App instance at runtime and provide a convenient way to test the behavior of warm/cold starts from pytest via modifying `RuntimeState.app`.

Happy to change it if there's a more preferable way to accomplish the same, but a class attribute seemed to be the simplest way to achieve this.